lib/Page.js

```diff
@@ -694,6 +694,41 @@ class Page extends EventEmitter {
   }
 
   /**
+   * @param {string} selector
+   * @param {!Array<string>} values
+   */
+  async select(selector, ...values) {
+    const handle = await this.$(selector);
+    console.assert(handle, 'No node found for selector: ' + selector);
+
+    await this.evaluate((element, values) => {
+      if (element.nodeName.toLowerCase() !== 'select')
+        throw new Error('Element is not a <select> element.');
+
+      const options = Array.from(element.options);
+
+      if (Array.isArray(values) !== true)
+        values = [values];
+
+      if (element.multiple !== true) {
+        element.value = values[0];
+
+        if (element.value !== values[0])
+          options.some(option => option.selected = values[0] === option.label);
+      } else {
+        options.forEach(option => {
+          option.selected = values.includes(option.value) || values.includes(option.label);
+        });
+      }
+
+      element.dispatchEvent(new Event('change'));
+      element.dispatchEvent(new Event('input'));
```

Comment:
do we need to dispatch 'input'? Nightmare gets away [without it](https://github.com/segmentio/nightmare/blob/07b4ba441c9836ab351d091f44e89af9fdf5eb88/lib/actions.js#L355)

Reply:
I think so, even listeners could only be listening to `input`, and I think those are also the events that the browser sends on a typical mouse-selection scenario.

Reply:
`input` is fired on select changes. So it should be fired to best reflect what browsers do.

Reply:
I'll keep dispatching `change` and `input` events, since that's what browsers do.